reana_db/models.py

```diff
@@ -513,9 +513,7 @@ def __init__(
         self.git_provider = git_provider
         self.restart = restart
         self._run_number = self.assign_run_number(run_number)
-        self.workspace_path = workspace_path or build_workspace_path(
-            self.owner_id, self.id_
-        )
+        self.workspace_path = workspace_path
```

Comment:
I think we want to keep the old code here, since we need to call `build_workspace_path()` once the workflow initializes